### PR TITLE
Some changes for building with the latest version of conan on OSX.

### PR DESCRIPTION
**Pull request by coding3d**
I was getting some errors on OSX. I was able to build the package with these modifications. I hope they are correct :)

### CODE REVIEW

File: conanfile.py [hunk 1]
```diff
@@ -17,6 +17,12 @@ class GTestConan(ConanFile):
     exports = "CMakeLists.txt"
     url="http://github.com/lasote/conan-gtest"
     license="https://github.com/google/googletest/blob/master/googletest/LICENSE"
+    
+    def config(self):
```
Comment (lasote) [inline conanfile.py]: Gtest library is C++ so this setting compiler.libcxx is needed.

File: test/conanfile.py [hunk 2]
```diff
@@ -9,10 +9,16 @@ class DefaultNameConan(ConanFile):
     settings = "os", "compiler", "arch", "build_type"
     generators = "cmake"
     requires = "gtest/1.7.0@lasote/stable"
+    
+    def config(self):
```
Comment (lasote) [inline test/conanfile.py]: Idem

**Comment by lasote** [comment]
Thanks @coding3d! try to remove the config method (the setting compiler.libcxx is needed because gtest is a c++ library) and see what happens with CI ok?

**Comment by coding3d** [comment]
I did but it crashed. I'll send more details after my next iteration.
Sometimes it takes a while before I get a chance to work on my project.
Thanks though!

**Comment by coding3d** [comment]
I have removed the config method and, as a matter of fact, it seems to work now. I am not sure why it produced an error before... I have upgraded my conan version in the meantime, so maybe it had something to do with that... I'll try another pull request.

**Pull request closed without merging**